conf/messages

```diff
@@ -106,19 +106,21 @@ locked.account = You''ve made too many unsuccessful attempts to sign in and your
 
 # Questionnaire error messages:
 
-error.required.gender = You must tell us your gender identity or choose prefer not to say.
-error.required.sexOrientation = You must tell us your sexual orientation or choose prefer not to say.
-error.required.ethnicity = You must tell us your ethnic group or choose prefer not to say.
+error.required.gender = You must tell us your gender identity or choose prefer not to say
+error.required.sexOrientation = You must tell us your sexual orientation or choose prefer not to say
```

Comment:
The choose prefer not to say should perhaps be in quotes. it is on line 117, 118, etc